graspologic/nominate/VNviaSGM.py

```diff
@@ -0,0 +1,391 @@
+import numpy as np
+from ..match import GraphMatch as GMP
+from sklearn.base import BaseEstimator
+import itertools
+import warnings
+
+
+class VNviaSGM(BaseEstimator):
+    """
+    This class implements Vertex Nomination via Seeded Graph Matching
+    (VNviaSGM) with the algorithm described in [1].
+
+    VNviaSGM is a nomination algorithm, so instead of completely matching two
+    graphs `A` and `B`, it proposes a nomination list of potential matches in
+    graph `B` to a vertex of interest (VOI) in graph `A` with associated
+    probabilities. VNviaSGM generates an initial induced subgraph about the
```

Comment:
```suggestion
    Rather than providing a 1-1 matching for the vertices of two
    graphs, as in :class:`~graspologic.match.GraphMatch`, VNviaSGM ranks the potential matches for a vertex
    in one to graph to the vertices in another graph, based on probability of matching.
```
move the second paragraph as is under the references as Notes, and replace with this